keyboards/waterfowl/readme.md

```diff
@@ -0,0 +1,16 @@
+# Waterfowl
+
+The Waterfowl is a 36-key split keyboard inspired by the Kyria and Arch-36.
+
+* Keyboard Maintainer: [CyanDuck](https://github.com/JW2586)
+* Hardware Supported: Pro Micro 5V/16MHz and compatible.
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Could this be filled in?